commands.py

```diff
@@ -160,9 +161,7 @@ def get_colour_name(requested_colour):
 
-        def RGB(r, g, b, colour):
-            h, s, l = RGB_to_HSL(r, g, b)
-            c, m, y, k = RGB_to_CMYK(r, g, b)
+        def color_embed(colour, r, b, g, c, m, y, k, h, s, l):
```

Comment:
All the colours had similar rendering so I pulled the rendering into its own function.

Reply:
is there a need to update the docstring to match the updated matching criteria? or nah